files/es/learn/css/css_layout/floats/index.md

```diff
@@ -93,8 +93,6 @@ Si guardas y recargas la página, verás algo parecido a lo que esperarías: la
 
 Ahora, si guardas y recargas, podrás ver algo parecido a lo siguiente:
 
```

Comment:
That was there to fix how the `embedlivesample` macro takes the codes and take only the current example and no the code from the whole document, maybe putting a texto to te title

```suggestion
## Ejemplo

```